BuildPropertiesPlugin/plugin/src/main/groovy/com/novoda/buildproperties/BuildProperties.groovy

```diff
@@ -0,0 +1,41 @@
+package com.novoda.buildproperties
+
+class BuildProperties {
+
+    final String name
+    File file
+    Properties entries
+
+    BuildProperties(String name) {
+        this.name = name
+    }
+
+    String getName() {
```

Comment:
Just a styling question. These getters and setters are not necessary in Groovy. Did you want them to be explicit?